common/Cargo.toml

```diff
@@ -49,6 +49,7 @@ tracing = { workspace = true }
 utoipa = { workspace = true, features = ["url"] }
 uuid = { workspace = true, features = ["v5", "serde"] }
 walker-common = { workspace = true, features = ["bzip2", "liblzma"]}
+humantime = "2.1.0"
```

Comment:
`humantime = { workspace = true }`, right?

If you need that specific version, add it in `../Cargo.toml`